src/utils/utils.js

```diff
@@ -352,6 +352,30 @@ const getRequestToken = (reqToken) => {
     return token;
 };
 
+const createAuthToken = (user) => {
+    const payload = {
+        user: {
+            id: user.id,
+            email: user.email,
+            firstname: user.firstname,
+            lastname: user.lastname,
+            username: user.username,
+            roles: user.roles,
+            privilege: getUserPrivilege(user),
+            isverified: user.isverified,
+        },
+    };
+    const exp = keys.authExpTime;
+    // create a token
+    const token = jwt.sign(payload, keys.privateKey, {
+        expiresIn: exp, // in seconds
+        issuer: keys.iss,
+        algorithm: 'RS256',
+    });
+
+    return token;
+};
+
```

Comment:
Please see the `createJWTCookie` method in utils.js. [Link](https://github.com/devclub-iitd/SingleSignOn/blob/3358d88abdaa7202a5921609fb8009b345c0c209/src/utils/utils.js#L40).

Instead of creating a separate method for token creation, split the `createJWTCookie` method into two methods - First for token creation (similar to this `createAuthToken`), second for cookie creation (the latter part of `createJWTCookie`). Use the `tokenName` argument to choose the expiry time of the token.